### ISSUE TITLE
Additional support for the Cam slim 2

### ISSUE THREAD

**Issue by SiGregory1973** [AlexxIT/SonoffLAN#1512]
I can view live video with the generic camera integration but is there a possibility of getting the human detection notification?
Would be nice to use this as a trigger (much like what a Google nest camera does)

**Comment by SiGregory1973** [comment]
I guess so. You can have it send you notifications and also use it as a trigger in a scene.

**Comment by cypherbits** [comment]
I would like this too. It has an alarm that can be manually triggered.

**Comment by igi8819** [comment]
+1

**Comment by andregoncalvespires** [comment]
It would be great to be able to adjust the camera settings in addition to having the other sensors.

**Comment by jms92100** [comment]
This cam is cheap and has many good features. 
Currently we can get the rtsp flow and .. thats all . The sonoff integration gives a sensor wich is not usefull . 
Could you instead provide :
- a switch to put the cam in sleep mode or awaking it . That will allow the detection when the cam is on and avoid it when the cam is off
- if possible an info wich gives the move detection status
- a switch to put the alarm on and off

I would like to integrate this with alarmo that will be excellent
Thanks for taking attention to my comment

**Comment by gabrielgaspardev** [comment]
+1

**Comment by MarcLeinenDE** [comment]
+1

**Comment by nicolasbeltran-creator** [comment]
+1

Any developments? Tx

**Comment by jms92100** [comment]
Yes , it supports onvif now.
I did add switches connected to ha in order to manage it.
It is still the cheapest cam you can get

Cordialement

**Comment by AlexxIT** [comment]
I once bought a two Tapo cameras on sale for $10 each. RTSP, ONVIF, Two way audio...

**Comment by nicolasbeltran-creator** [comment]
@jms92100 Yes, I updated the firmware through the eWelink app and ONVIF became available. Motion detection states now available on HA.

Merci!

I will check the Tapo cameras, for now the Sonoff Slim Gen2 seems a good deal.

**Comment by claorden** [comment]
I tried with ONVIF but only the Sensors do appear:

<img width="386" height="344" alt="Image" src="https://github.com/user-attachments/assets/b7483cbb-c652-41d2-b025-d089d4dcef41" />

It would be great to have access to other options (mainly sounding the alarm).
